app/interactions/interactions.validators.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const _ = require('lodash');
+const validation = require('../components/validation.js');
+const commonSchemas = require('../common/schemas.js');
+
+const schemas = {
+
+  listInteractions: {
+    'type': 'object',
+    'properties': Object.assign({}, commonSchemas.pagingAndSortingProperties, commonSchemas.searchingProperties)
+  },
+
+  createInteraction: {
+    'type': 'object',
+    'properties': {
+      'time': commonSchemas.datetime,
+      'notes': commonSchemas.varchar(65535)
```

Comment:
how much is it Jo? i made it into 65535 because of the maximum length of text in mysql